src/stores/one.ts

```diff
@@ -75,6 +78,8 @@ export const useOneStore = defineStore('one', (): OneState => {
 
   const isLoading = shallowRef(false)
   const isOpen = shallowRef(false)
+  const mobileMenu = shallowRef(false)
+  const mobileBreakpoint = { value: shallowRef(false) }
```

Comment:
This doesn't seem right. It would require you to do **mobileBreakpoint.value.value**.